docs/react-migration-guide.md

```diff
@@ -1,6 +1,208 @@
 # React hooks upgrade / migration guide
 
-## Version 2.x to 3.x
+## Version 1.x to 2.x
+
+### ChannelProvider
+
+In previous versions, you were able to provide channel options as a parameter in the `useChannel`/`usePresence` hooks. This often led to errors when using hooks with the same channel name but different options, or when attempting to dynamically change options for a channel.
```

Comment:
i think slightly softer language might make this sound less scary 😅 
```suggestion
In previous versions, you were able to provide channel options as a parameter in the `useChannel`/`usePresence` hooks. This could, in some cases, lead to errors when using hooks with the same channel name but different options, or when attempting to dynamically change options for a channel.
```

Reply:
Agree 😅